src/components/Footer/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { ReactNode } from 'react'
+
+import {
+  Box,
+  Flex,
+  Link,
+  SimpleGrid,
+  Stack,
+  Text,
+  useColorModeValue,
+} from '@chakra-ui/react'
+import { motion } from 'framer-motion'
+import NextImage from 'next/image'
+import { useTranslation } from 'react-i18next'
+
+import { Container } from '@components'
+
+import { FooterNav } from '../FooterNav'
+import { SocialButtons } from '../SocialButtons'
+
+export const ListHeader = ({ children }: { children: ReactNode }) => {
+  return (
+    <Text fontWeight={'500'} fontSize={'lg'} mb={2}>
+      {children}
+    </Text>
+  )
+}
+
+export const Footer = (): JSX.Element => {
+  const { t } = useTranslation()
+  return (
+    <Box
+      bg={useColorModeValue('gray.700', 'gray.900')}
+      color={useColorModeValue('primary.600', 'primary.900')}
```

Comment:
`useColorModeValue` is being used when you switch between light and dark modes.

We are not using darkMode in this project. So it's not necessary to use this hook.